util/kvcache/simple_lru_test.go

```diff
@@ -104,6 +107,11 @@ func (s *testLRUCacheSuite) TestPut(c *C) {
 
 		root = root.Next()
 	}
+
+	// test dropped key-value pair
+	for i := 0; i < len(droppedKv); i++ {
+		c.Assert(droppedKv[keys[i]], Equals, vals[i])
+	}
```

Comment:
If `len(droppedKv) == 0`, the upper test is useless.

How about test like this:
```go
diff --git a/util/kvcache/simple_lru_test.go b/util/kvcache/simple_lru_test.go
index 96e7bba0c..6feb416a2 100644
--- a/util/kvcache/simple_lru_test.go
+++ b/util/kvcache/simple_lru_test.go
@@ -76,10 +76,12 @@ func (s *testLRUCacheSuite) TestPut(c *C) {
        c.Assert(lru.size, Equals, uint(3))

        // test for non-existent elements
+       c.Assert(len(droppedKv), Equals, 2)
        for i := 0; i < 2; i++ {
                element, exists := lru.elements[string(keys[i].Hash())]
                c.Assert(exists, IsFalse)
                c.Assert(element, IsNil)
+               c.Assert(droppedKv[keys[i]], Equals, vals[i])
        }
```

Reply:
Thx, I forgot this, I'll add it ASAP.